M2/Macaulay2/e/schreyer-resolution/res-f4-m2-interface.cpp

```diff
@@ -427,7 +427,8 @@ MutableMatrix* ResF4toM2Interface::to_M2_MutableMatrix(SchreyerFrame& C,
   Nterm** comps = newarray(Nterm*, nrows);
   Nterm** last = newarray(Nterm*, nrows);
 
-  int* m1 = M->make_one();
+  monomial m1 = M->make_one();
+  // FIXME: is exp a monomial or exponent vector?
```

Comment:
I agree, but maybe I'll leave this for you and Frank to resolve because a few lines down the `to_expvector` function
```m2
C.ring().monoid().to_expvector(w, exp, comp);
```
Calls a method of `ResMonoidDense` in `schreyer-resolution/res-moninfo-dense.hpp`.